docs/code-frame.md

```diff
@@ -151,3 +179,49 @@ const result = codeFrameColumns(rawLines, location, {
 
 console.log(result);
 ```
+
+## Migrating from `@babel/highlight`
```

Comment:
Can you move this chapter before the `Upgrading from prior versions`, since that section was for Babel 6 to 7. We can probably remove that section for Babel 8.